nailgun/entities.py

```diff
@@ -1970,27 +1983,14 @@ def create_missing(self):
         super(LifecycleEnvironment, self).create_missing()
         if (self.name != 'Library' and  # pylint:disable=no-member
                 not hasattr(self, 'prior')):
-            response = client.get(
-                self.path('base'),
-                data={
-                    u'name': u'Library',
-                    # pylint:disable=no-member
-                    u'organization_id': self.organization.id,
-                },
-                **self._server_config.get_client_kwargs()
-            )
-            response.raise_for_status()
-            results = response.json()['results']
+            results = self.search({'organization'}, {u'name': u'Library'})
```

Comment:
> Could we use such statements here?:
> ```
> results = self.search({u'organization_id': self.organization}, {u'name': u'Library'})
> ```
> or
> ```
> results = self.search({u'organization_id': self.organization.id}, {u'name': u'Library'})
> ```

No. The first parameter to `search()`, `fields`, is a set of field names. Do not pass a dict where a set is expected.